games-simulation/openrct2/openrct2-0.2.4.ebuild

```diff
@@ -0,0 +1,182 @@
+		truetype? (
+			media-libs/fontconfig:=
+			media-libs/freetype:2=
+			media-libs/sdl2-ttf:=
+		)
+		twitch? (
+			net-misc/curl:=[ssl]
+			libressl? ( dev-libs/libressl:0= )
+			!libressl? ( dev-libs/openssl:0= )
+		)
+	)
+	multiplayer? (
+		libressl? ( dev-libs/libressl:0= )
+		!libressl? ( dev-libs/openssl:0= )
+	)
+"
+
+RDEPEND="
+	${DEPEND_RDEPEND}
+	dedicated? (
+		acct-group/openrct2
+		acct-user/openrct2
+	)
+"
+
+DEPEND="
+	${DEPEND_RDEPEND}
+	app-arch/unzip
+	test? ( dev-cpp/gtest )
+"
+
+BDEPEND="virtual/pkgconfig"
+
+RESTRICT="!test? ( test )"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.2.4-disable-desktop-and-icon-files-without-gui.patch"
+	"${FILESDIR}/${PN}-0.2.4-disable-optional-features.patch"
+	"${FILESDIR}/${PN}-0.2.4-include-additional-paths.patch"
+)
+
+src_unpack() {
+	unpack "${P}".tar.gz
+
+	mkdir -p "${S}"/data/title || die
+	cd "${S}"/data/title || die
+	unpack "${PN}-${MY_PN_TS}-${MY_PV_TS}".zip
+
+	mkdir -p "${S}"/data/object || die
+	cd "${S}"/data/object || die
+	unpack "${PN}-${MY_PN_OBJ}-${MY_PV_OBJ}".zip
+
+	if use test; then
+		mkdir -p "${S}"/testdata/replays || die
+		cd "${S}"/testdata/replays || die
+		unpack "${PN}-${MY_PN_RPL}-${MY_PV_RPL}".zip
+	fi
+}
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	# Don't treat warnings as errors.
+	sed -e 's/-Werror//' -i CMakeLists.txt || die
+}
+
+src_configure() {
+	# Note: There is currently no support for Disord-RPC and Google Benchmark,
+	# as both packages do not exist in Gentoo, so support for them has been disabled.
+	local mycmakeargs=(
+		-DDISABLE_DISCORD_RPC=ON
+		-DDISABLE_GOOGLE_BENCHMARK=ON
+		-DDISABLE_GUI=$(usex dedicated)
+		-DDISABLE_HTTP_TWITCH=$(usex !twitch)
+		-DDISABLE_NETWORK=$(usex !multiplayer)
+		$(usex !dedicated "-DDISABLE_OPENGL=$(usex !opengl)" "")
+		-DDISABLE_TTF=$(usex !truetype)
+		-DDOWNLOAD_OBJECTS=OFF
+		-DDOWNLOAD_REPLAYS=OFF
+		-DDOWNLOAD_TITLE_SEQUENCES=OFF
+		-DENABLE_LIGHTFX=$(usex lightfx)
+		-DPORTABLE=OFF
+		-DSTATIC=OFF
+		$(usex test "-DSYSTEM_GTEST=ON" "")
+		-DWITH_TESTS=$(usex test)
+		-DUSE_MMAP=ON
+	)
+
+	cmake-utils_src_configure
+}
+
+src_test() {
+	# Since the tests need the OpenRCT2 data,
+	# we need to copy them into the build directory,
+	# otherwise some tests will fail, as they don't find the OpenRCT2 data.
+	# It is currently not possible to override the path.
+	# See: https://github.com/OpenRCT2/OpenRCT2/issues/6473
+	cp -r "${S}"/data "${BUILD_DIR}" || die
+
+	cmake-utils_src_test
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	if use dedicated; then
+		newinitd "${FILESDIR}"/openrct2.initd openrct2
+		newconfd "${FILESDIR}"/openrct2.confd openrct2
+	fi
+}
+
+pkg_postinst() {
+	if [[ -z ${REPLACING_VERSIONS} ]]; then
+		einfo "OpenRCT2 needs the original RollerCoaster Tycoon 2 data files."
+		einfo ""
+		einfo "You need the folders 'Data', 'ObjData', 'Scenarios' and 'Tracks'"
+		einfo "from the original release and copy them into '/usr/share/openrct2/rct2'."
```

Comment:
There is a difference. The default scenarios are typically intended for the singleplayer. If you throw the scenario into the dedicated server, it could be modified with ongoing progress. But those default scenarios should not be modified in first place, as these are intended to be choosen by start of OpenRCT2 for single player. It's also not entirely correct, that someone will use even the default scenarios, as it will be more likely, that custom created scenarios are used. IMHO, there should be a clean separation between original data and (maybe custom) server data. And the server places all his files, including configuration in `/var/lib/openrct2`.

Reply:
If they are being modified, I agree that it should go to /var/lib.

Reply:
Yup, this can be the case. So will leave it as it is.